mpp/include/mpp_io_read.inc

```diff
@@ -659,67 +767,72 @@
                     allocate(i2vals(len), STAT=istat)
                     if ( istat .ne. 0 ) then
                        write(text,'(A)') istat
-                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta): Unable to allocate temporary array i2vals.  STAT = "&
-                            & //trim(text))
+                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta): "// &
+                       "Unable to allocate temporary array i2vals.  STAT = "//trim(text))
                     end if
                     error=NF_GET_ATT_INT2(ncid,i,trim(attname),i2vals);
                     call netcdf_err( error, mpp_file(unit), attr=mpp_file(unit)%Axis(dimid)%att(j) )
                     mpp_file(unit)%Axis(dimid)%Att(j)%fatt(1:len)=i2vals(1:len)
                     if( verbose .and. pe == 0  ) &
-                         print *, 'AXIS ',trim(mpp_file(unit)%Axis(dimid)%name),' ATT ',trim(attname),' ',mpp_file(unit)&
-                         &%Axis(dimid)%Att(j)%fatt
+                         print *, 'AXIS ',trim(mpp_file(unit)%Axis(dimid)%name),' ATT ',&
+                                & trim(attname),' ',mpp_file(unit)%Axis(dimid)%Att(j)%fatt
                     deallocate(i2vals)
                  case (NF_INT)
                     allocate(mpp_file(unit)%Axis(dimid)%Att(j)%fatt(len), STAT=istat)
                     if ( istat .ne. 0 ) then
                        write(text,'(A)') istat
-                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta): Unable to allocate mpp_file%Axis%Att%fatt, "//&
-                            & "NF_INT CASE.  STAT = "//trim(text))
+                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta):"// &
+                       " Unable to allocate mpp_file%Axis%Att%fatt, "// &
+                       "NF_INT CASE.  STAT = "//trim(text))
                     end if
                     allocate(ivals(len), STAT=istat)
                     if ( istat .ne. 0 ) then
                        write(text,'(A)') istat
-                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta): Unable to allocate temporary array ivals.  STAT = "&
-                            & //trim(text))
+                       call mpp_error(FATAL, "mpp_io_mod(mpp_read_meta): "// &
+                       "Unable to allocate temporary array ivals.  STAT = "//trim(text))
```

Comment:
Since this is within a string, probably should probably use an & right in front of the continued line as well. Same for the other of the instances in the file.